superset/commands/database/resync_permissions.py

```diff
@@ -0,0 +1,276 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
```

Comment:
I was initially going to keep both `ResyncPermissionsCommand` and `ResyncPermissionsAsyncCommand` in the same file, but I was facing a circular import as `ResyncPermissionsAsyncCommand` would import the celery task and the celery task imports `ResyncPermissionsCommand`. I can centralize both in the same file and then perform a local import in the celery task if preferred.

Another option is to unify this in a single command with a flag (like `async_mode`) to make it DRYer, and then handle the differences at the `validate` method. 

Open to suggestions here.

Reply:
Alright! Let me work on this